woocommerce/woocommerce-android
# Enable Custom Amounts M1 feature

Closes: #10085 


### Description

Enable Custom Amounts M1 for the release version and add release notes

### Testing instructions

Build a release build and ensure you can see the Custom amounts feature in the order creation screen.
